programming_foundations/lean_fridge_app_final_proj/lean_fridge_setup.py

```diff
@@ -0,0 +1,32 @@
+import os
```

Comment:
can you move all of these files into another subdirectory called "db" ? also do they need to be under the prog foundations course ? are they also needed for some assignment in that course?